src/tests/8_conversation.test.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from "chai";
+import { Conversation } from "../db/models/conversation.model";
+import ConversationService from "../db/service/ConversationService";
+
+describe("Conversation service can", () => {
+	let conversationService: ConversationService;
+	let mockedConversation: Conversation;
+	it("be setup", () => {
+		conversationService = new ConversationService();
+		expect(conversationService).not.equal(undefined);
+	});
+
+	it("create a new converation", async () => {
+		let response: any;
+		try {
+			response = await conversationService.createConversation(["user1", "user2"]);
+		} catch (error) {
+			console.log(error);
+		}
```

Comment:
it does not make sense to have try-catch in test, unless you expect your server to crash. Move this `try-catch` to the code instead